notes/style-guide.md

```diff
@@ -206,30 +216,29 @@ This is very much a work-in-progress and is not exhaustive.
 #### Preconditions and postconditions
 
 * Preconditions should only be included in names of results if
-  "important" (mostly judgement call).
+  "important" (mostly judgment call).
 
 * Preconditions of results should be prepended to a description
   of the result by using the symbol `⇒` in names (e.g. `asym⇒antisym`)
 
 * Preconditions and postconditions should be combined using the symbols
   `∨` and `∧` (e.g. `m*n≡0⇒m≡0∨n≡0`)
 
-* Try to avoid the need for bracketing but if necessary use square
+* Try to avoid the need for bracketing, but if necessary use square
   brackets (e.g. `[m∸n]⊓[n∸m]≡0`)
 
 #### Operators and relations
 
 * Operators and relations should be defined using mixfix notation where
-  applicable (e.g. `_+_`, `_<_`)
+  applicable (e.g. `_+_`, `_<_`). In a name containing one or more `_`, each underscore can be interpreted as the place where each argument goes.
```

Comment:
Again, I think this is a general point about Agda rather than something that belongs in the style guide.

Reply:
I just wanted to give more context to not expert users. An alternative could be linking [mixfix notation](https://agda.readthedocs.io/en/latest/language/mixfix-operators.html)?

Reply:
Yes, I like that idea. A hyperlink on the word `misfix` to Agda's misfix notation page at `ReadTheDocs` should do nicely.